src/plugins/gui/providers/paybisProvider.ts

```diff
@@ -64,6 +65,7 @@ const asApiKeys = asObject({
 const asPaymentMethodId = asValue(
   'method-id-credit-card',
   'method-id-credit-card-out',
+  'method-id_bridgerpay_revolutpay',
```

Comment:
What's the rule for `-` vs `_`?

Reply:
I don't know. It's Paybis and what they want to call these values. We have no control.